packages/backend/src/services/HealthService/HealthService.ts

```diff
@@ -46,9 +45,8 @@ export class HealthService extends BaseService {
             await this.migrationModel.getMigrationStatus();
 
         if (!isComplete) {
-            throw new UnexpectedDatabaseError(
-                'Database has not been migrated yet',
-                { currentVersion },
+            console.warn(
+                `Database migrations are not up to date. Current version: ${currentVersion}`,
```

Comment:
Are we ok to keep this as a `warn` for devx @owlas ? 
I can remove it completely, but I'm thinking this could be useful for debugging 🤔

Reply:
Useful!